tests/testthat/test-toolGetMapping.R

```diff
@@ -6,8 +6,12 @@ test_that("toolGetMapping works", {
                              CountryCode = c("ABW", "AFG", "AGO","AIA", "ALA", "ALB"), 
                              RegionCode = c("LAM", "OAS", "SSA", "LAM","EUR", "NEU")), 
                         row.names = c(NA, 6L), class = "data.frame")
+  write.csv(expected,file = "expected.csv",quote = FALSE,row.names = FALSE)
```

Comment:
CRAN policy requires that these files which are written, must be stored in a temporary directory

Reply:
not sure how to deal with this issue.. is this file written in a temporary directory? https://github.com/pik-piam/madrat/blob/9a41765c7cc595b6a98070ed0ce5cc7fa4ba775b/tests/testthat/test-calcOutput.R#L133

Reply:
yes, it is. You can see that a few lines above (mainfolder = tempdir())
in your case you can solve it easily by using tempdir() or tempfile() to use a temporary directory where you put the file or directly a temporary file (if the file name does not matter)